### PR TITLE
SAM-1200 Increase column size for descriptions and comments in some fields

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

**Comment by jonespm** [comment]
This needs review. Both for the Oracle scripts and that I'm actually increasing the right columns. I really don't 100% know if the correct,incorrect and general feedback all maps into the comments column or not yet.

File: reference/docs/conversion/sakai_11_oracle_conversion.sql [hunk 1]
```diff
@@ -641,3 +641,37 @@ create index idx_qrtz_ft_tg on qrtz_fired_triggers(SCHED_NAME,TRIGGER_GROUP);
...
+--
+-- KNL-1200 Oracle conversion, size increases of Samigo columns
+--
+alter table SAM_PUBLISHED_ASSESSMENT_T add tempcol clob;
+update SAM_PUBLISHED_ASSESSMENT_T set tempcol=description;
+alter table SAM_PUBLISHED_ASSESSMENT_T drop column description;
+alter table SAM_PUBLISHED_ASSESSMENT_T rename column tempcol to description;
+
+alter table SAM_PUBLISHEDSECTION_T add tempcol clob;
+update SAM_PUBLISHEDSECTION_Tset tempcol=description;
```
Comment (juanjmerono) [inline reference/docs/conversion/sakai_11_oracle_conversion.sql]: You miss one space here "SAM_PUBLISHEDSECTION_Tset" :+1:

Comment (jonespm) [inline reference/docs/conversion/sakai_11_oracle_conversion.sql]: Thanks, fixed in most recent commit

**Comment by jonespm** [comment]
Updated commit to resolve conflicts, also put the conversions into a 12 conversion file (which doesn't exist yet)

**Comment by jonespm** [comment]
Tested this out all I can locally, this I'm just going to merge this for 12 and see how nightly comes up with it.

**Pull request merged**